tools/slintpad/src/editor_widget.ts

```diff
@@ -551,37 +551,38 @@ class EditorPaneWidget extends Widget {
     protected async handle_lsp_url_request(
         era: number,
         url: string,
-    ): Promise<string> {
+    ): Promise<[string, null | number]> {
         if (this.#url_mapper === null) {
-            return Promise.resolve("Error: Can not resolve URL.");
+            return Promise.resolve(["Error: Can not resolve URL.", null]);
```

Comment:
So we resolve it, but it's an error? Seems wrong to me.

Reply:
Still open :-)

Reply:
Done now.